force-app/main/default/lwc/propertyTile/__tests__/propertyTile2.test.js

```diff
@@ -0,0 +1,70 @@
+import { createElement } from 'lwc';
```

Comment:
Can you rename the test file name to something more meaningful.
How about `propertyTitle.small.test.js`?